### PR TITLE
Add Expeditor::Service#reset_status!

### PR DESCRIPTION
**Pull request by makimoto**
Implement "2. Status reset" of #8.
`Expeditor::Service#reset_status!` re-generates the service's bucket and service status is reset.

@cookpad/dev-infra 👓 ?

### CODE REVIEW

File: lib/expeditor/service.rb [hunk 1]
```diff
@@ -9,11 +9,11 @@ def initialize(opts = {})
       @threshold = opts.fetch(:threshold, 0.5) # is 0.5 ok?
       @non_break_count = opts.fetch(:non_break_count, 100) # is 100 ok?
       @sleep = opts.fetch(:sleep, 1)
-      bucket_opts = {
+      @bucket_opts = {
         size: 10,
         per: opts.fetch(:period, 10).to_f / 10
       }
-      @bucket = Expeditor::Bucket.new(bucket_opts)
+      @bucket = Expeditor::Bucket.new(@bucket_opts)
```
Comment (eisuke) [inline lib/expeditor/service.rb]: Should it just call `reset_status!`?

Comment (makimoto) [inline lib/expeditor/service.rb]: Fixed in 18ffbb6e80a870e9235b35f1b3f4306b961f51c4

**Comment by eisuke** [comment]
LGTM

**Comment by k0kubun** [comment]
LGTM

**Pull request merged**